gateway_code/rest_server.py

```diff
@@ -310,7 +310,7 @@ def _cond_route(self, obj, func, path, *route_args, **route_kwargs):
 
     def route(self, path, method='GET', callback=None, *args, **kwargs):
         """Add a route but catch some exceptions."""
-        # pylint:disable=arguments-differ
+        # pylint:disable=arguments-differ, keyword-arg-before-vararg
```

Comment:
maybe we shouldnt disable the warning but address the *args, *kwargs used in all these _route methods in GatewayRest ? Most of them are not really needed IMO